{"@t":"2020-03-25T11:57:54.2719324Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Booting Umbraco 8.5.5.","TimingId":"3386860","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:54.2938729Z","@mt":"Booting site '{HostingSiteName}', app '{HostingApplicationID}', path '{HostingPhysicalPath}', server '{MachineName}'.","HostingSiteName":"AwesomeUmbracoProject","HostingApplicationID":"/LM/W3SVC/2/ROOT","HostingPhysicalPath":"C:\\Users\\mathi\\Desktop\\AwesomeUmbracoProject\\AwesomeUmbracoProject\\","MachineName":"DESKTOP-114L7VB","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:54.4444823Z","@mt":"Acquiring.","SourceContext":"Umbraco.Core.MainDom","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:54.4454438Z","@mt":"Acquired.","SourceContext":"Umbraco.Core.MainDom","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:55.3433388Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Resolving composer types.","TimingId":"6e1a2f4","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:55.5144454Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Resolved.","Duration":170,"TimingId":"6e1a2f4","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:56.0091218Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:56.0101193Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:56.0131106Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:56.6630410Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from local cache file","TimingId":"b3725e7","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:56.7288669Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":65,"TimingId":"b3725e7","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:56.8015516Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from local cache file","TimingId":"29229e3","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:56.8015516Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":0,"TimingId":"29229e3","SourceContext":"Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:57.1067402Z","@mt":"Initializing Umbraco internal event handlers for cache refreshing.","SourceContext":"Umbraco.Web.Cache.DistributedCacheBinderComponent","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:57.1566146Z","@mt":"Adding examine event handlers for {RegisteredIndexers} index providers.","RegisteredIndexers":3,"SourceContext":"Umbraco.Web.Search.ExamineComponent","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:57.2683915Z","@mt":"Starting initialize async background thread.","SourceContext":"Umbraco.Web.Search.BackgroundIndexRebuilder","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:57.2693878Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Booted.","Duration":3008,"TimingId":"3386860","SourceContext":"Umbraco.Core.Runtime.CoreRuntime","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":1,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":1,"HttpRequestId":"06bfc5ed-aba3-487a-8022-b1c59c7b3e6a"}
{"@t":"2020-03-25T11:57:57.7376617Z","@mt":"New url {Url} detected, re-discovering application url.","Url":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"526bec4a-c865-471e-8dfb-1c6303dd3a8c"}
{"@t":"2020-03-25T11:57:57.7386527Z","@mt":"ApplicationUrl: {UmbracoAppUrl} (UmbracoModule request)","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Core.Sync.ApplicationUrlHelper","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":2,"HttpRequestId":"526bec4a-c865-471e-8dfb-1c6303dd3a8c"}
{"@t":"2020-03-25T12:00:39.6513306Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username mathias.vanner@live.dk from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"72936beb-c023-44bf-99f2-8d1559c7fd3f"}
{"@t":"2020-03-25T12:00:39.6533317Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: mathias.vanner@live.dk logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":3,"HttpRequestId":"72936beb-c023-44bf-99f2-8d1559c7fd3f"}
{"@t":"2020-03-25T12:01:30.8805392Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:01:38.4532215Z","@mt":"Login attempt succeeded for username {Username} from IP address {IpAddress}","Username":"mathias.vanner@live.dk","IpAddress":"::1","SourceContext":"Umbraco.Core.Security.UmbracoMembershipProviderBase","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":14,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpSessionId":"zmwideivpmypcfbjskxfnpxa","HttpRequestNumber":4,"HttpRequestId":"d302fe51-6936-40b4-a8a4-af9307af79c8"}
{"@t":"2020-03-25T12:01:51.3240571Z","@mt":"Scheduled health check results:","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3240571Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Folder & File Permissions","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3240571Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All folders have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3240571Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Folder & File Permissions","HealthCheckResult":"Success","HealthCheckMessage":"All files have the correct permissions set.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"SMTP Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"SMTP Settings","HealthCheckResult":"Error","HealthCheckMessage":"In the Web.config file system.net/mailsettings section, the host is not configured.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Macro errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Macro errors","HealthCheckResult":"Error","HealthCheckMessage":"MacroErrors are set to 'throw' which will prevent some or all pages in your site from loading completely if there are any errors in macros. Rectifying this will set the value to 'inline'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Notification Email Settings","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Notification Email Settings","HealthCheckResult":"Error","HealthCheckMessage":"Notification email is still set to the default value of <strong>your@email.here</strong>.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Try Skip IIS Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Try Skip IIS Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Try Skip IIS Custom Errors is set to 'true' and you're using IIS version '10.0'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Custom Errors","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Custom Errors","HealthCheckResult":"Success","HealthCheckMessage":"Custom errors are set to 'RemoteOnly'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' all completed successfully.","HealthCheckName":"Trace Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Trace Mode","HealthCheckResult":"Success","HealthCheckMessage":"Trace mode is disabled.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Debug Compilation Mode","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Debug Compilation Mode","HealthCheckResult":"Error","HealthCheckMessage":"Debug compilation mode is currently enabled. It is recommended to disable this setting before go live.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cross-site scripting Protection (X-XSS-Protection header)","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Der kunne ikke oprettes forbindelse til fjernserveren'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Der kunne ikke oprettes forbindelse til fjernserveren'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Content/MIME Sniffing Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Content/MIME Sniffing Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Der kunne ikke oprettes forbindelse til fjernserveren'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Excessive Headers","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Excessive Headers","HealthCheckResult":"Warning","HealthCheckMessage":"[httpsCheckInvalidUrl]","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"Click-Jacking Protection","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"Click-Jacking Protection","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL http://localhost/umbraco - 'Der kunne ikke oprettes forbindelse til fjernserveren'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Checks for '{HealthCheckName}' completed with errors.","@l":"Warning","HealthCheckName":"HTTPS Configuration","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"WARN "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"You are currently not viewing the site using the HTTPS scheme.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Info","HealthCheckMessage":"The appSetting 'Umbraco.Core.UseHttps' is set to 'false' in your web.config file. Once you access this site using the HTTPS scheme, that should be set to 'true'.","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:01:51.3250724Z","@mt":"Result for {HealthCheckName}: {HealthCheckResult}, Message: '{HealthCheckMessage}'","HealthCheckName":"HTTPS Configuration","HealthCheckResult":"Error","HealthCheckMessage":"Error pinging the URL https://localhost/umbraco - 'Der kunne ikke oprettes forbindelse til fjernserveren'","SourceContext":"Umbraco.Web.HealthCheck.HealthCheckResults","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO "}
{"@t":"2020-03-25T12:06:34.8986642Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:10:55.7411108Z","@mt":"Error getting remote dashboard data from {UrlPrefix}{Url}","@l":"Error","@x":"System.Net.Http.HttpRequestException: Response status code does not indicate success: 404 (Not Found).\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Editors.DashboardController.<GetRemoteXml>d__5.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Editors\\DashboardController.cs:linje 171","UrlPrefix":"https://umbraco.com/","Url":"feeds/videos/members","SourceContext":"Umbraco.Web.Editors.DashboardController","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":33,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR","HttpRequestNumber":5,"HttpRequestId":"7005eafb-02ab-48a5-847a-7d95041c4c05"}
{"@t":"2020-03-25T12:11:38.9110820Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:16:42.9273919Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:21:46.9496886Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:26:50.9615721Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:30:13.0958747Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: mathias.vanner@live.dk, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":12,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":6,"HttpRequestId":"09152058-6cc0-43b0-a4f3-790ac4cc7197"}
{"@t":"2020-03-25T12:31:55.0102198Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:36:59.0269473Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:42:03.0357465Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:47:07.0591985Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:52:11.0831550Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T12:57:15.1038769Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:02:19.1228668Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:07:23.1516344Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:12:27.1873923Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:17:31.2143552Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:22:35.2315103Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:27:39.2547775Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":13,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:32:43.2750723Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:37:47.2948637Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":21,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:42:51.3088468Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:47:55.3307655Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:52:59.3476237Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T13:58:03.3649807Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:03:07.3830095Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:08:11.4238851Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:13:15.4474234Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:18:19.4550337Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:23:23.4662007Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:28:27.4838525Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:33:31.5029466Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:38:35.5274722Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:43:39.5505969Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:48:43.5664832Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:53:47.5861648Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T14:58:51.6095615Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:03:55.6285970Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:08:59.6482378Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":9,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:14:03.6697562Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:19:07.6952317Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":17,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:24:11.7177475Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:29:15.7298906Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:34:19.7512384Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:39:23.7761457Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:44:27.7867348Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:49:31.8168519Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":25,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:54:35.8284600Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T15:59:39.8506833Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:04:43.8706950Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":27,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:09:47.8921779Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":21,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:14:51.9110311Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":24,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:19:55.9227378Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:24:59.9349588Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:30:03.9588165Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:35:07.9760498Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:40:11.9892579Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:45:16.0080496Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:50:20.0317161Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T16:55:24.0378102Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":22,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:00:28.0547344Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":21,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:05:32.0892013Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:10:36.0995822Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":43,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:15:40.1079814Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":42,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:20:44.1563715Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":41,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:24:30.3595105Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"Login attempt succeeded for username mathias.vanner@live.dk from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"91d22f4d-eda0-48d2-a831-22c12977d8c2"}
{"@t":"2020-03-25T17:24:30.3595105Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User: mathias.vanner@live.dk logged in from IP address ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":5,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":7,"HttpRequestId":"91d22f4d-eda0-48d2-a831-22c12977d8c2"}
{"@t":"2020-03-25T17:25:48.1785474Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":40,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:30:52.1963344Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":38,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:35:56.2144475Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":40,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:41:00.2248904Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":44,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:44:00.1511382Z","@mt":"[{EventType}] Event Id: {EventId}, State: {State}","EventType":"Information","EventId":0,"State":"User logged will be logged out due to timeout: mathias.vanner@live.dk, IP Address: ::1","SourceContext":"Umbraco.Core.Logging.OwinLogger","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":10,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"INFO ","HttpRequestNumber":8,"HttpRequestId":"d11de988-1880-4226-9676-9e8b6affa94b"}
{"@t":"2020-03-25T17:46:04.2438831Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":44,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:51:08.2687018Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":44,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T17:56:12.2833293Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":51,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:01:16.3023885Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":51,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:06:20.3194527Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:11:24.3350191Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:16:28.3503390Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:21:32.3691561Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:26:36.3795757Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:31:40.3974210Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":51,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:36:44.4103181Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":56,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:41:48.4329032Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":56,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:46:52.4442623Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":56,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:51:56.4625854Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T18:57:00.4752666Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":55,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
{"@t":"2020-03-25T19:02:04.4964208Z","@mt":"Keep alive failed (at '{UmbracoAppUrl}').","@l":"Error","@x":"System.Net.Http.HttpRequestException: Der opstod en fejl under afsendelse af anmodningen. ---> System.Net.WebException: Der kunne ikke oprettes forbindelse til fjernserveren ---> System.Net.Sockets.SocketException: Der kunne ikke oprettes forbindelse, fordi destinationscomputeren aktivt n√¶gtede det 127.0.0.1:80\r\n   ved System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)\r\n   ved System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ved System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)\r\n   --- Slut p√• staksporing af indre undtagelser ---\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   ved System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   ved Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__4.MoveNext() i D:\\a\\1\\s\\src\\Umbraco.Web\\Scheduling\\KeepAlive.cs:linje 63","UmbracoAppUrl":"http://localhost/umbraco","SourceContext":"Umbraco.Web.Scheduling.KeepAlive","ProcessId":16540,"ProcessName":"iisexpress","ThreadId":45,"AppDomainId":2,"AppDomainAppId":"LMW3SVC2ROOT","MachineName":"DESKTOP-114L7VB","Log4NetLevel":"ERROR"}
